src/components/BugBountyCards.js

```diff
@@ -108,149 +108,120 @@ const Text = styled.div`
-          </li>
-        </ul>
-      </Text>
-      <Divider />
-      <SubHeader>
-        <Translation id="page-eth2-bug-bountycard-subheader-2" />
-      </SubHeader>
-      <Text>
-        <Translation id="page-eth2-bug-bountycard-text-2" />
-      </Text>
-      <StyledButton to="https://forms.gle/Gnh4gzGh66Yc3V7G8">
-        <Translation id="page-eth2-bug-bountycard-high-risk" />
-      </StyledButton>
-    </Card>
-    <Card>
-      <CriticalLabel>
-        <Translation id="page-eth2-bug-bountycard-label-7" />
-      </CriticalLabel>
-      <H2>
-        <Translation id="page-eth2-bug-bountycard-critical" />
-      </H2>
-      <Description>
-        <Translation id="page-eth2-bug-bountycard-label-8" />
-      </Description>
-      <Divider />
-      <SubHeader>
-        <Translation id="page-eth2-bug-bountycard-subheader" />
-      </SubHeader>
-      <Text>
-        <ul>
-          <li>
-            <Translation id="page-eth2-bug-bountycard-li-8" />
-          </li>
-        </ul>
-      </Text>
-      <Divider />
-      <SubHeader>
-        <Translation id="page-eth2-bug-bountycard-subheader-2" />
-      </SubHeader>
-      <Text>
-        <Translation id="page-eth2-bug-bountycard-text-3" />
-      </Text>
-      <StyledButton to="https://forms.gle/Gnh4gzGh66Yc3V7G8">
-        <Translation id="page-eth2-bug-bountycard-critical-risk" />
-      </StyledButton>
-    </Card>
+    {bugBountyCardsInfo.map((card, idx) => (
+      <Card key={`bug-bountycard-${idx}`}>
+        {card.lowLabelTranslationId && (
+          <LowLabel>
+            <Translation id={card.lowLabelTranslationId} />
+          </LowLabel>
+        )}
+        {card.mediumLabelTranslationId && (
+          <MediumLabel>
+            <Translation id={card.mediumLabelTranslationId} />
+          </MediumLabel>
+        )}
+        {card.highLabelTranslationId && (
+          <HighLabel>
+            <Translation id={card.highLabelTranslationId} />
+          </HighLabel>
+        )}
+        {card.criticalLabelTranslationId && (
+          <CriticalLabel>
+            <Translation id={card.criticalLabelTranslationId} />
+          </CriticalLabel>
+        )}
+        <H2>
+          <Translation id={card.h2TranslationId} />
+        </H2>
+        <Description>
+          <Translation id={card.descriptionTranslationId} />
+        </Description>
+        <Divider />
+        <SubHeader>
+          <Translation id={card.subHeader1TranslationId} />
+        </SubHeader>
+        <Text>
+          <ul>
+            <li>
+              <Translation id={card.textLi1TranslationId} />
+            </li>
+            {card.textLi2TranslationId && (
+              <li>
+                <Translation id={card.textLi2TranslationId} />
+              </li>
+            )}
+            {card.textLi3TranslationId && (
+              <li>
+                <Translation id={card.textLi3TranslationId} />
+              </li>
+            )}
+          </ul>
```

Comment:
Would it maybe be better to have an array within the object and map over that instead of using a conditional here?

```suggestion
          <ul>
            {card.severityList.map((listItemId) => (
              <li>
                <Translation id={listItemId} />
              </li>
            ))}
          </ul>
```